packages-common/pluggable-widgets-tools/tests/commands.spec.js

```diff
@@ -93,14 +93,14 @@ describe("pluggable-widgets-tools commands", () => {
         if (platform === "native") {
             it("tests originally fail due to snapshots and are fixed with '-u'", async () => {
                 process.stderr.write(`[${widgetName}] Testing unit tests...\n`);
-                await execFailedAsync("npm test", workDir);
```

Comment:
This will break for old packages

Reply:
Old packages have `test:unit` as well

Reply:
I re-enabled npm test though and moved coverage check to the correct place.